modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppPistacheServerCodegen.java

```diff
@@ -81,8 +81,6 @@ public CppPistacheServerCodegen() {
 
         reservedWords = new HashSet<>();
 
-        supportingFiles.add(new SupportingFile("modelbase-header.mustache", "model", modelNamePrefix + "ModelBase.h"));
```

Comment:
if you remove them from the code, then also remove the mustache files themselves

Reply:
Removed modelbase-header.mustache and modelbase-source.mustache.